src/Estimators/OneBodyDensityMatrices.cpp

```diff
@@ -11,7 +11,7 @@
 // File refactored from: QMCHamiltonians/DensityMatrices1B.cpp
 //////////////////////////////////////////////////////////////////////////////////////
 
-
+#include "Concurrency/OpenMP.h"
```

Comment:
According to @ye-luo the first header in a cpp should always be the matching header.  I've also been following this convention for some time. It is described perhaps somewhat unclearly in the development guide part of the manual.  That said I'm planning on dropping the omp dependency from this file since appears to be a legacy debugging relic.

Reply:
Use `#pragma omp master` instead of `omp_get_thread_num() == 0` and you don't need the header file.